src/platform/forms-system/src/js/components/BackLink.jsx

```diff
@@ -70,12 +70,12 @@ const BackLink = ({ router, routes, location, form, setData }) => {
   }
 
   return (
-    <div className="vads-u-margin-top--2 vads-u-margin-bottom--4">
+    <nav className="vads-u-margin-top--2 vads-u-margin-bottom--4">
```

Comment:
Just out of curiosity, what's the benefit of `nav` vs `div` here?

Reply:
I think it has to do with screen reader regions, and how a screen reader can jump around by regions. This was an issue that Jeana had opened and requested to change.